src/scripting/flash/text/textrenderer.cpp

```diff
@@ -0,0 +1,74 @@
+/**************************************************************************
+    Lightspark, a free flash player implementation
+
+    Copyright (C) 2009-2013  Alessandro Pignotti (a.pignotti@sssup.it)
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**************************************************************************/
+
+#include "scripting/flash/text/textrenderer.h"
+#include "scripting/class.h"
+#include "scripting/argconv.h"
+#include <iomanip>
+
+using namespace lightspark;
+
+void TextRenderer::sinit(Class_base* c)
+{
+	CLASS_SETUP_NO_CONSTRUCTOR(c, ASObject, CLASS_SEALED | CLASS_FINAL);
+
+	c->setDeclaredMethodByQName("setAdvancedAntiAliasingTable","",Class<IFunction>::getFunction(c->getSystemState(),_setAdvancedAntiAliasingTable),GETTER_METHOD,false);
```

Comment:
according to spec setAdvancedAntiAliasingTable is a normal method, not a getter

Reply:
Still have the problem

Reply:
Thanks for the reply. Have been fiddling with the code a bit hence why GETTER_METHOD got stuck in there

Reply:
Still have "setAdvancedAntiAliasingTable is not a function". Everything else works though